### PR TITLE
Make mypy checking strict, add py.typed

### PR DESCRIPTION
**Pull request by sportsracer**

### CODE REVIEW

File: setup.cfg [hunk 1]
```diff
@@ -2,4 +2,14 @@
...
 
 [mypy]
+strict = True
+disallow_subclassing_any = False
+
+[mypy-jsonschema.*]
+ignore_missing_imports = True
+
+[mypy-sentry_sdk.*]
+ignore_missing_imports = True
```
Comment (harpaj) [inline setup.cfg:12]: this should not be required, the Sentry SDK is typed: https://github.com/getsentry/sentry-python/blob/master/sentry_sdk/py.typed

**Review by harpaj — COMMENTED** [review]
Very nice :)  Would be nice to check this automatically with Github Actions, but no clue how complex that is to set up.

File: retwist/util/sentry.py [hunk 2]
```diff
@@ -1,12 +1,12 @@
...
+from typing import Any, Dict
 
 import sentry_sdk
 from twisted.python import log
 from twisted.python.failure import Failure
 
 
 def add_request_context_to_scope(context, scope):
-    # type: (Dict, sentry_sdk.Scope) -> None
+    # type: (Dict[Any, str], sentry_sdk.Scope) -> None
```
Comment (cristianc-ty) [inline retwist/util/sentry.py:9]: The context is Dict[str, Any].

**Review by cristianc-ty — APPROVED** [review]

**Comment by sportsracer** [comment]
> Very nice :) Would be nice to check this automatically with Github Actions, but no clue how complex that is to set up.

Got this working, too :)

https://github.com/trustyou/retwist/actions/runs/501534838

https://github.com/trustyou/retwist/blob/master/.github/workflows/python-package.yml

**Pull request merged**